templates/sponsors/admin/preview-statement-of-work.html

```diff
@@ -0,0 +1,283 @@
+{% extends "easy_pdf/base.html" %}
+{% load humanize %}
+
+{% block extra_style %}
+    <style type="text/css">
+        .dynamic {
+            background-color: #dddddd;
+        }
+
+        .center {
+            text-align: center;
+        }
+
+        .paragraph {
+            text-indent: 25px;
+        }
+
+        .items-list {
+            padding-left: 50px;
+            margin-left: 25px;
+        }
+
+        .items-list ol {
+            list-style-type: lower-alpha;
+            padding-left: 10px;
+        }
+
+        .items-list ol li .item-title {
+            font-weight: normal;
+        }
+
+        .item-title {
+            font-weight: bold;
+            text-decoration: underline;
+        }
+
+        #mailing-info {
+            padding-left: 0px;
+        }
+
+        #mailing-info * {
+            margin: 0;
+        }
+
+        #signatures-section {
+            margin-left: 250px;
+        }
+
+        #signatures-section p, #signatures-section span {
+            margin: 0;
+        }
+
+        #signatures-section .legend {
+            text-indent: 20px;
+        }
+
+        #exhibit-list {
+            padding-left: 25px;
+        }
+
+        #exhibit-list ol {
+            padding-left: 100px;
+        }
+
+        body {
+            font-family: "Times New Roman";
+            text-align: justify;
+            text-justify: inter-word;
+            font-size: 120%;
+        }
+
+        @media print {
+          .pb {page-break-after: always;} /* page break */
+        }
+    </style>
+{% endblock %}
+
+{% block content %}
+<h1 class='center'>SPONSORSHIP AGREEMENT</h1>
+
+<p class='paragraph'><b>THIS SPONSORSHIP AGREEMENT</b> (the <b>“Agreement”</b>) is entered into and made
+effective as of the {{ start_date|date:"dS" }} day of {{ start_date|date:"F, Y"}} (the <b>“Effective Date”</b>), by and between
+Python Software Foundation (the <b>“PSF”</b>), a Delaware nonprofit corporation, and <b>{{ sponsor.name|upper }}</b> (<b>“Sponsor”</b>), a {{ sponsor.state }} corporation. Each of the PSF and Sponsor are
+hereinafter sometimes individually referred to as a <b>“Party”</b> and collectively as the <b>“Parties”</b>.</p>
+
+<h1 class='center'>RECITALS</h1>
+
+<p class='paragraph'><b>WHEREAS</b>, the PSF is a tax-exempt charitable organization (EIN 04-3594598) whose
+mission is to promote, protect, and advance the Python programming language, and to support
+and facilitate the growth of a diverse and international community of Python programmers (the
+<b>“Programs”</b>);</p>
+
+<p class='paragraph'><b>WHEREAS</b>, Sponsor is {{ sponsor.description }}; and</p>
```

Comment:
I don't think we want the "description" here.

Reply:
@ewdurbin I don't have the link for the sample contract I based myself when coding this, but there was some dynamic information that should be displayed here. From what we currently have in the `Sponsor` model, the `description` attribute sounded as a good fit. Maybe are we missing this input in our models?